docs/iterators.md

```diff
@@ -54,16 +51,17 @@ let iteratorResult3 = frame.next(); //{ done: false, value: Component { name: 'l
 let iteratorResult4 = frame.next(); //{ done: false, value: Component { name: 'right' } }
 let iteratorResult5 = frame.next(); //{ done: true }
 
-//It is possible to access the value of iterator result via the value property:
+//Yineleyici sonucunun değerine 'value' özelliği aracılığıyla ulaşmak mümkündür.
 let component = iteratorResult1.value; //Component { name: 'top' }
 ```
-Again. Iterator itself is not a TypeScript feature, this code could work without
-implementing Iterator and IteratorResult interfaces explicitly.
-However it is very helpful to use these common
-ES6 [interfaces](./types/interfaces.md) for code consistency.
 
-Ok, Nice, but could be more helpful. ES6 defines the *iterable protocol*
-which includes [Symbol.iterator] `symbol` if Iterable interface implemented:
+Tekrar belirtmek gerekirse yineleyici bir TypeScript özelliği değildir.  Bu kod Iterator ve IteratorResult arayüzlerini açıkça uygulamadan da çalışabilir.
```

Comment:
implement'in turkcesi uygulamak ama sanki burada anlami kaybediyoruz gibi geldi. okudugumda bir sey anlayamadigim icin ingilizcesini okuma ihtiyaci hissettim :(